crates/zed/build.rs

```diff
@@ -56,5 +57,17 @@ fn main() {
             "cargo:rustc-link-arg-bins=/MANIFESTINPUT:{}",
             manifest.canonicalize().unwrap().display()
         );
+
+        let mut res = winresource::WindowsResource::new();
```

Comment:
Why is this copy right section included?

Reply:
The copyright information set here is displayed when you right-click on the `Zed.exe` file and view its properties. I noticed that other software like VSCode, Steam, Chrome, and others all have this information, so I included it. If you feel it's too early to add it now, it can be removed without affecting the program's operation.

Reply:
I think we'd rather not have this in this PR.

Reply:
Done.